content/images/socialmedia/github.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="512px" height="516px" viewBox="0 0 512 516" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Nice these are the red one right? I do not see them yet included. We can try adding them in the `SOCIAL` dict maybe